### PR TITLE
test fixes

### PR DESCRIPTION
**Pull request by nayimovag**
fixed tests [C353233](https://foliotest.testrail.io/index.php?/cases/view/353233), [C353956](https://foliotest.testrail.io/index.php?/cases/view/353956), [C356817
](https://foliotest.testrail.io/index.php?/cases/view/356817), [C357034
](https://foliotest.testrail.io/index.php?/cases/view/357034)

### CODE REVIEW

File: cypress/downloads/matchedRecords_841.1323884058409370.csv [hunk 1]
```diff
@@ -0,0 +1,2 @@
+User name,User id,External system id,Barcode,Active,Type,Patron group,Departments,Proxy for,Last name,First name,Middle name,Preferred first name,Email,Phone,Mobile phone,Date of birth,Addresses,Preferred contact type id,Enrollment date,Expiration date,Created date,Updated date,Tags,Custom fields
```
[the remark this answers is not in the document]
Comment (nayimovag) [reply cypress/downloads/matchedRecords_841.1323884058409370.csv]: fixed

File: cypress/fixtures/bulkEditImport_615.7123718526583370.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+User name,User id,External system id,Barcode,Active,Type,Patron group,Departments,Proxy for,Last name,First name,Middle name,Preferred first name,Email,Phone,Mobile phone,Date of birth,Addresses,Preferred contact type id,Enrollment date,Expiration date,Created date,Updated date,Tags,Custom fields
```
[the remark this answers is not in the document]
Comment (nayimovag) [reply cypress/fixtures/bulkEditImport_615.7123718526583370.csv]: fixed

File: cypress/fixtures/userUUIDs_773.7428841154592370.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+30d0cc2d-b394-4625-8d06-6a7a94b4ccdd
```
[the remark this answers is not in the document]
Comment (nayimovag) [reply cypress/fixtures/userUUIDs_773.7428841154592370.csv]: fixed

**Review by penfriend — APPROVED** [review]

**Comment by penfriend** [comment]
jenkins build: https://jenkins-aws.indexdata.com/job/Testing/job/Cypress%20tests/626/allure/#suites . all autotests are passed

**Pull request merged**